src/metpy/plots/cartopy_utils.py

```diff
@@ -74,4 +74,4 @@ class CartopyStub:
 
     def __getattr__(self, item):
         """Raise an error on any attribute access."""
-        raise RuntimeError(f'CartoPy is required to use this feature ({item}).')
+        raise AttributeError(f'CartoPy is required to use this feature ({item}).')
```

Comment:
Slightly unrelated, but while you're in here do you want to standardize this (and update the test) to be the same as in #1928, ie `f'Cannot use {item} without Cartopy installed.'`?

Reply:
Sure, got that now.